ignite/cmd/chain.go

```diff
@@ -60,13 +72,79 @@ chain.
 		Args:    cobra.ExactArgs(1),
 	}
 
-	c.AddCommand(
-		NewChainServe(),
-		NewChainBuild(),
-		NewChainInit(),
-		NewChainFaucet(),
-		NewChainSimulate(),
-	)
+	c.AddCommand(addConfigMigrationVerifier(NewChainServe()))
+	c.AddCommand(addConfigMigrationVerifier(NewChainBuild()))
+	c.AddCommand(addConfigMigrationVerifier(NewChainInit()))
+	c.AddCommand(addConfigMigrationVerifier(NewChainFaucet()))
+	c.AddCommand(addConfigMigrationVerifier(NewChainSimulate()))
 
 	return c
 }
+
+func addConfigMigrationVerifier(cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().AddFlagSet(flagSetConfig())
+	cmd.Flags().AddFlagSet(flagSetYes())
```

Comment:
I suggest to move those flag adds in the parent command (`NewChain()` function), using `cmd.PersistentFlags()` so they are propagated to the sub-commands (again :P)

Reply:
Changed in ede171be9c3a6951489c2206944b4bf483f85b80